jekyll/_cci2/server/air-gapped-installation/phase-2-configure-object-storage.adoc

```diff
@@ -0,0 +1,105 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 2 - Configuring Object Storage
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to configure object storage through Min.IO to run CircleCI Server in an air-gapped environment.
+:icons: font
+:toc: macro
+:toc-title:
+
+[#configure-minio]
+== Initial Min.IO Setup
+
+[#create-airgap-data-bucket]
+=== 1. Create a airgap-data bucket.
```

Comment:
This bucket is never referenced again, and I am not sure why the values.yaml would go there.  I think i can be removed